pkgs/development/compilers/haxe/default.nix

```diff
@@ -78,7 +105,7 @@ let
         description = "Programming language targeting JavaScript, Flash, NekoVM, PHP, C++";
         homepage = "https://haxe.org";
         license = with licenses; [ gpl2 bsd2 /*?*/ ];  # -> docs/license.txt
-        maintainers = [ maintainers.marcweber ];
+        maintainers = [ maintainers.marcweber maintainers.locallycompact ];
```

Comment:
```suggestion
        maintainers = with maintainers; [ marcweber locallycompact ];
```

Reply:
Huh?

Reply:
This `with` bindings are a matter of taste, really, and change nothing semantically. Also not changing this keeps the diff more readable and saves me the trouble of rebasing again.